invalid_sample_generator.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import csv
+import xml.etree.ElementTree as ET
+import json
+import random
+import os
+
+
+biosample_field_vals = {
+    "collection date":"collection_date"
+}
+
+mandatory_error_file_extension = '_mandatory.json'
```

Comment:
I would save the files in a folder by the error type, rather than add something to the name.